draft-dejong-remotestorage-06.txt

```diff
@@ -9,13 +9,12 @@ Expires: 3 June 2016                                  30 November 2015
 
 Abstract
 
-    This draft describes a protocol by which client-side applications,
-    running inside a web browser, can communicate with a data storage
-    server that is hosted on a different domain name. This way, the
-    provider of a web application need not also play the role of data
-    storage provider. The protocol supports storing, retrieving, and
-    removing individual documents, as well as listing the contents of an
-    individual folder, and access control is based on bearer tokens.
+    This draft describes a protocol how applications can discover 
```

Comment:
well, it depends on how you look at it. I also think it should be there, but mentioning that gives the impression that other uses are not envisioned :) I'd also really like to see it as a generic storage server that one day has desktop/mobile sync!